fpdcc/ccfp-asset-dashboard
# Fix missing geo calculations

## Overview
- closes #265 

Includes a Heroku review app config change to support a different database connection (to be addressed in another PR, but related to #252)

### Notes

This happened due to two things:
1. for the `Zone` model, the import was skipping the zones, so I changed it to an `update_or_create`
2. for districts, moving self.save() outside of the loop fixed this...dunno why

## Testing Instructions
- visit the review app: https://ccfp-asset-d-feature-di-1nlzy2.herokuapp.com/
- navigate to a project with some assets ([here's one](https://ccfp-asset-d-feature-di-1nlzy2.herokuapp.com/projects/428/)). see how the projects don't include the districts/zones
- add/remove an asset. visit the project overview page and see how the districts/zones get updated
